### PR TITLE
Add RSS feed

### PR DESCRIPTION
**Pull request by faultables**
In this update, I add functionality to generate RSS Feed to your blog, because I love RSS (and probably people love it too!).

There is no additional configuration (on end) for this change, it works seamlessly with your current configuration (especially on CI-side).

This change won't add more bundle size to your site, well honestly it will add this line:

```html
<link
  rel="alternate"
  type="application/rss+xml"
  title="adriantirusli"
  href="/rss.xml"
/>
```

To every your minified `.html` page, minified. This is how it look like on RSS Client/Reader:

Screen Shot 2020-03-26 at 4 45 18 AM

I use [ReadKit](https://readkitapp.com/) for this.

And this how it look like on [Google FeedBurner](https://feedburner.google.com/) just to make sure the structures are rendered *correctly* across clients since FeedBurner is from Google.

Screen Shot 2020-03-26 at 4 49 43 AM

The RSS Feed is generated on `build` time, so you don't need to change anything — beside apply this change to the upstream 😉

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ require("dotenv").config({
   path: `.env.${activeEnv}`,
 })
 
+let siteUrl = `https://adriantirusli.me/blog/`
```
Comment (faultables) [inline gatsby-config.js]: Also, keep in mind with this line since `siteMetadata` don't give us any `siteUrl` or equivalent node.

Comment (faultables) [reply gatsby-config.js]: Fixed on https://github.com/adriantirusli/adriantirusli.me/pull/1/commits/ae4df85431d587bcee984b60f51968a6ffa4018d!

**Comment by faultables** [comment]
https://deploy-preview-1--punyaorang.netlify.com/rss.xml

**Comment by adriantirusli** [comment]
Great!
Thanks a lot

**Pull request merged**